src/2024h2/candidates.md

```diff
@@ -73,6 +73,7 @@ In many cases the work being described will definitely happen, but it is not cle
 | [impl trait everywhere][ITE]     | [oli-obk]         | [Lang], [Types]    |
 | [Patterns of empty types][PET]   | [Nadrieril]       | [Lang], [Compiler] |
 | [Relaxing the Orphan Rule][RTOR] | [Josh-Triplett][] | [Lang]             |
+| [Cargo Script][CS]               | [epage]           | [Cargo]            |
```

Comment:
The instructions [said](https://rust-lang.github.io/rust-project-goals/how_to/propose_a_goal.html)

> Edit 2024h2/slate.md to add your goal into the "WIP" table, following the existing examples.

but the WIP sections were just links so I'm guessing this is what the latest version of what was intended.

Reply:
Thanks :) yeah, that's a bit outdated...